Fundamentals/Design/Rendering-Media/index-v9.md

```diff
@@ -0,0 +1,116 @@
+
+### Example 2: Accessing a Media Image Modelsbuilder item based upon its Id
+
+As with example one, we are accessing a MediaType `image` using the same ID assumption.
+
+```csharp
+@using ContentModels = Umbraco.Web.PublishedModels;
+@{
+    // Since the Image Model generated by Modelsbuilder is a compatible type to IPublishedContent we can use the 'as' operator to convert into the ModelsBuilder Umbraco.Web.PublishedModels.Image class
+    var mediaItem = Umbraco.Media(1055) as ContentModels.Image;
+}
+@if (mediaItem!=null)
+{
+    // you could add this as an extension method to the Umbraco.Web.PublishedModels.Image class
+    var orientationCssClass = mediaItem.UmbracoWidth > mediaItem.UmbracoHeight ? "img-landscape" : "img-portrait";
+
+    <img src="@mediaItem.Url()" alt="@mediaItem.Name" class="@orientationCssClass" />
+}
+```
+
+:::note
+It's always worth having Null checks around your code when retrieving media, in case the conversion fails or Media() returns null. This makes your code more robust.
+:::
+
+### Other Media Items such as `File`
+
+Accessing other media items can be performed in the same way, the techniques aren't limited to the `Image` type, but it is one of the most common use cases.
+
+## Image Cropper
+
+Image Cropper can be used with `Image` media types and is the default option for the umbracoFile property on an `Image` media type.
+
+When working with the ImageCropper for an image the `GetCropUrl` extension method is used to retrieve, cropped, resized versions of the uploaded image. Details of the Image Cropper property editor and other examples of using it can be found [here](../../Backoffice/Property-Editors/Built-in-Property-Editors/Image-Cropper.md). The following example is a quick example to help you get started.
+
+### Example of using Image Cropper with the Models Builder strongly typed `Image` model
+
+```csharp
+@using ContentModels = Umbraco.Web.PublishedModels;
+
+@{
+    // We can use the 'as' operator to convert between IPublishedContent and ModelsBuilder Umbraco.Web.PublishedModels.Image
+    var mediaItem = Umbraco.Media(1234) as ContentModels.Image;
+}
+@if (mediaItem!=null)
+{
+    <img src="@mediaItem.GetCropUrl("myCropAlias")" alt="@mediaItem.Name" />
+}
+```
+
+This example assumes that you have set up a crop called **myCropAlias** on your Image Cropper data type.
+
+If you want the original, uncropped image, you can ignore the GetCropUrl extension method and use one of the previously discussed approaches as shown below.
+
+```html
+<img src="@mediaItem.Url" />
+```
+
+### More information
+
+- [Media Picker](../../Backoffice/Property-Editors/Built-in-Property-Editors/Media-Picker/index.md)
+- [Image Cropper](../../Backoffice/Property-Editors/Built-in-Property-Editors/Image-Cropper.md)
+- [Creating a Media Type](../../Data/Creating-Media/index.md#creating-a-media-type)
```

Comment:
Until the release these links should link to the V9 articles if they exist :)

Reply:
@jonat123 Only the Media Picker article contains the v9 version - its updated now :)